src/plugins/orangehrmAdminPlugin/test/fixtures/testcases/ModulesAPITestCases.yaml

```diff
@@ -13,7 +13,8 @@ GetAll:
       performance: true,
       maintenance: false,
       mobile: false,
-      directory: true
+      directory: true,
+      claim: true
```

Comment:
```suggestion
```
this should pass without sending `claim` param. but response body will contain `claim` param.